### PR TITLE
API Days Recap post

### PR DESCRIPTION
**Pull request by benjagm**
API Days Recap post

**GitHub Issue:** https://github.com/json-schema-org/community/issues/528

**Summary**: After an amazing experience as community partner at API Days Paris, this is the recap post with some insights for our community.

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

No

### CODE REVIEW

File: pages/blog/posts/apidays-paris-2023-recap.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+<div className='flex flex-wrap justify-center items-center gap-4 w-full'>
+    <img className='max-w-400 px-20' src='/img/posts/2023/apidays-paris-2023-recap/booth-apidays.jpeg'/>
+</div>
+
+## Meaningful conversations with organizations
+At JSON Schema, we need sponsors to help us deliver our Roadmap, and this event enabled some great opportunities that hopefully will turn into future sponsorships.
+
+## Learning from the AsyncAPI and the OpenAPI conference tracks
+At JSON Schema, we have been discussing organizing a JSON Schema conference, but doing it alone will require a lot of resources and some years to get its own status. Instead, we returned inspired by great OpenAPI and AsyncAPI tracks, which we think is our path forward!. We'd love to host our own track in the future!
```
Comment (Relequestual) [inline pages/blog/posts/apidays-paris-2023-recap.md:46]:
```suggestion
At JSON Schema, we have been discussing organizing a JSON Schema conference, but doing it alone will require a lot of resources and some years to get its own status. Instead, we returned inspired by great OpenAPI and AsyncAPI tracks, which we think is our path forward! We'd love to host our own track in the future!
```

File: pages/blog/posts/apidays-paris-2023-recap.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Connecting with JSON Schema and API practitioners
+It has been great having the chance to connect with JSON Schema practitioners in person. We received great feedback and created great connections to follow up with.
+
+<div className='flex flex-wrap justify-center items-center gap-4 w-full'>
+    <img className='max-w-400 px-20' src='/img/posts/2023/apidays-paris-2023-recap/booth-apidays.jpeg'/>
+</div>
+
+## Meaningful conversations with organizations
+At JSON Schema, we need sponsors to help us deliver our Roadmap, and this event enabled some great opportunities that hopefully will turn into future sponsorships.
```
Comment (Relequestual) [inline pages/blog/posts/apidays-paris-2023-recap.md:43]: This is good. I would also like to mention that we are not JUST looking for sponsorship in terms of money, but in terms of "people resource". We want to connect with stakeholders / those invested in JSON Schema and its ecosystem. We want those stakeholders to be more directly part of our community, and be involved in driving our roadmap, like you say.

I don't know how you want to phrase that. I will leave that up to you.

**Review by Relequestual — CHANGES_REQUESTED** [review]
Overall this looks great. I have made one suggestion in terms of what I would like us communicate.

File: pages/blog/posts/apidays-paris-2023-recap.md (hunk 2 above)
Comment (benjagm) [reply pages/blog/posts/apidays-paris-2023-recap.md]: Great point. Thanks Ben. I improved this section to address this in my last push.

**Pull request merged**